testing/test_aux.out

```diff
@@ -47,16 +47,31 @@ RESULT skip security.passwords.http Port 80 is not open on target device.
 RESULT skip security.passwords.https Port 443 is not open on target device.
 RESULT skip security.passwords.telnet Port 23 is not open on target device.
 RESULT skip security.passwords.ssh Port 22 is not open on target device.
-RESULT fail security.passwords.http Default passwords have not been changed.
-RESULT fail security.passwords.https Default passwords have not been changed.
-RESULT fail security.passwords.telnet Default passwords have not been changed.
-RESULT fail security.passwords.ssh Default passwords have not been changed.
+RESULT skip security.passwords.http Port 80 is not open on target device.
+RESULT skip security.passwords.https Port 443 is not open on target device.
+RESULT skip security.passwords.telnet Port 23 is not open on target device.
+RESULT skip security.passwords.ssh Port 22 is not open on target device.
 RESULT pass security.passwords.http Default passwords have been changed.
 RESULT pass security.passwords.https Default passwords have been changed.
 RESULT pass security.passwords.telnet Default passwords have been changed.
 RESULT pass security.passwords.ssh Default passwords have been changed.
 RESULT skip security.firmware Could not retrieve a firmware version with nmap. Check bacnet port.
 RESULT pass security.firmware version found: ?\xFF\xFF\x19,>u\x08\x00no
+RESULT fail connection.dhcp_long 
+RESULT fail connection.min_send 
+RESULT info communication.type.broadcast 
+RESULT fail protocol.app_min_send 
```

Comment:
why are all these fail?  Where's the tests for pass or skip?

Reply:
Hi Trevor,

I've only tried to made code changes with regards to the communication.type.broadcast test - since there's been more frequent changes to the network module, I've added the network module back into test_aux.sh - but it's been removed for a long while so I'm not sure why the others are all failing or whether they were always like this, but I'll look into this.

Reply:
These tests rely on reading monitor_scan.pcap which is currently turned off within by test_aux.sh which might explain the fail. The setting might need to be changed

@em-redstone to note that connection.dhcp_long will need to be removed from the module (will form part of a different test module). The same for app_min_send as it's not part of the test plan

Reply:
Thanks @noursaidi, I can remove these two tests in the next commit if that's what you meant, or did you mean to just make note of it in the readme.md?

Reply:
@em-redstone I was noting it in case you were going to try and get the aux tests to pass/fail for them. We can do a seperate issue/PR to remove them

Reply:
I think we should keep them as is (exposed) and then fix them in another PR.  Before they were just hidden which is the worst situation. Now we can at least see what is wrong!  I'll send a note to the team to figure out how to address this.